awx/ui/src/screens/TopologyView/Legend.js

```diff
@@ -70,9 +71,7 @@ function Legend() {
       <DescriptionList isHorizontal isFluid>
         <DescriptionListGroup>
           <DescriptionListTerm>
-            <Button variant="primary" isSmall>
-              {t`C`}
-            </Button>
+            <Button isSmall>{t`C`}</Button>
```

Comment:
Just thinking about how this is going to be translated.  I think we should add an aria-label here to help with accessibility.